packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -85,20 +86,43 @@ export const getStaticProps = async () => {
 
 ### About Tabs
 
-The Tabs component allows a user to flip through multiple views within the same page.
+Tabs are labeled controls that allow users to switch between multiple views within a page. Tabs do not take the user to a new URL. To take users to a new URL, use [In Page Navigation](/components/in-page-navigation).
+
+Panels within a set of Tabs should present different views of similar, parallel information. For example, [these Twilio docs](https://www.twilio.com/docs/usage/api) contain a block of code on the right side of the page with Tabs, allowing users to view the code in different languages. [Reddit also uses tabs](https://www.reddit.com/t/technology/), switching the user’s view between **Posts**, **Communities**, and **Related Topics**. The Tabs in both of these examples control views; they don’t direct to a new page. Conversely, [the Github website](https://github.com/twilio-labs/paste) demonstrates how some interactions may seem like Tabs, but are actually in-page navigation. Clicking on one of the sections at the top, like **Code** or **Pull Requests**, changes the URL.
+
+Having a distinction between the two components (Tabs and In Page Nav) leads to an improved user experience by visually communicating that interacting will lead to different behaviors. It's also important for accessibility to select the correct component to use based on your use case, as using the correct component gives users of assistive technologies a better idea of where the interaction will lead.
+
+Choose Tabs when users don’t need to see content from multiple views simultaneously. When thinking through what type of content to put in your Tabs, remember that the content should be parallel across the board. What lives in one Tab should be a different view of what lives in another Tab.
+
+_Source: [Nielsen Norman Group](https://www.nngroup.com/articles/tabs-used-right/)_
+
+The Tabs component contains the following elements:
+
+- Tabset: a composition of a Tablist and Tabpanels
+- Tablist: a collection of Tabs
+- Tab: clickable element that causes a selection within the Tabset
+- Tabpanel: contains the content associated with the selected Tab
```

Comment:
```suggestion
- `Tab`: Clickable element that causes a selection within a tabset
- `TabPanel`: Contains the content associated with the selected `Tab`
- `TabList`: A collection of `Tab`s
- `TabPanels`: A collection of `TabPanel`s
- Tabset (or `Tabs`): Composition of `TabList` and `TabPanels`
```

so that Tab and Tabpanel are defined first before they're used as the sole definition of Tabset and Tablist.